scripts/install_third_party_libs.py

```diff
@@ -49,7 +49,7 @@ def install_prerequisite(package: Tuple[str]) -> None:
     Raises:
         Exception. If the package fails to install due to issues with --prefix.
     """
-    package_name, version_number, target_path = package
+    (package_name, version_number, target_path) = package
```

Comment:
Why is this changed?

Reply:
When I started doing mypy checks, it was throwing an error due to this line not being in parentheses; however, I realize now that that was actually due to package not being annotated with Tuple[str, str, str], but rather with Tuple[str]. Reverted.

Reply:
Done.